ansible/roles/master/templates/kubelet.j2

```diff
@@ -4,14 +4,11 @@
 # The address for the info server to serve on (set to 0.0.0.0 or "" for all interfaces)
 KUBELET_ADDRESS="--address=0.0.0.0"
 
-# The port for the info server to serve on
-# KUBELET_PORT="--port=10250"
-
 # You may leave this blank to use the actual hostname
-KUBELET_HOSTNAME="--hostname-override={{ inventory_hostname }}"
+KUBELET_HOSTNAME=""
 
-# location of the api-server
-KUBELET_API_SERVER="--api-servers=http://localhost:{{ kube_master_insecure_port }}"
+# location of the api-server. Set to none for unregistered master kubelet
+KUBELET_API_SERVER=""
 
 # Add your own!
-KUBELET_ARGS="--register-node=false --config={{ kube_manifest_dir }}"
+KUBELET_ARGS="--register-node=false --register-schedulable=false --config={{ kube_manifest_dir }} --allow-privileged=true"
```

Comment:
OK. We should be able to include the masters > 1 conditional logic to manage the vars.